internals/secrethub/migrate_config_envfile.go

```diff
@@ -39,7 +39,7 @@ func (cmd *MigrateConfigEnvfileCommand) Run() error {
 		return err
 	}
 
-	output, replaceCount, err := migrateTemplateTags(string(inFileContents), refMapping, "%s")
+	output, replaceCount, err := migrateTemplateTags(string(inFileContents), refMapping, "{{ %s }}")
```

Comment:
With this changed, I'm guessing that the last arg can be removed now from the `migrateTemplateTags` function signature entirely. Could you check?

Reply:
Actually, never mind my comment: this change should be reverted, because we don't want curly braces to end up in the `.env` file.